src/Nova/Block.php

```diff
@@ -26,9 +26,9 @@ class Block extends BaseResource
 
     /** @psalm-suppress UndefinedClass */
     protected array $filterClassList = [
-        \Tipoff\Scheduling\Filters\FutureBlocks::class,
-        \Tipoff\Scheduling\Filters\SlotRoomLocation::class,
-        \Tipoff\Scheduling\Filters\SlotRoom::class,
+        \Tipoff\Bookings\Filters\FutureBlocks::class,
+        \Tipoff\Bookings\Filters\SlotRoomLocation::class,
+        \Tipoff\Bookings\Filters\SlotRoom::class,
```

Comment:
@drewroberts here is another example that I have to call the Booking class

Reply:
Let's leave those 3 filters in this scheduling package

Reply:
The 3 filters that should be moved to tipoff/bookings are:

- Slot
- SlotDay
- FutureSlots